### PR TITLE
Nar-Sie will now corrupt airlocks, tables, windows, and windoors

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscadd: Nar-Sie will now corrupt airlocks, tables, windows, and windoors.
rscadd: Corrupted airlocks have no access restrictions.
tweak: Nar-Sie no longer causes destruction while moving around in favor of additional corruption.
/:cl:

Removes some signatures from nar-sie code.
new-sie

### CODE REVIEW

**Comment by Iamgoofball** [comment]
need some hellish totally-metal computer frames tbqh

**Comment by Cheridan** [comment]
i'll see about getting you some culty sprites for consoles, closets, and extinguisher cabinets

**Comment by Zilenan91** [comment]
See if you can get extinguisher cabinets to look like screaming mouths

**Comment by Anonmare** [comment]
That's pretty metal tbh fam

File: code/modules/power/singularity/narsie.dm [hunk 1]
```diff
@@ -52,23 +53,9 @@
...
-	if(isobj(A))
-		var/obj/O = A
-		O.ex_act(1)
-		if(O) qdel(O)
-
-	else if(isturf(A))
-		var/turf/T = A
-		T.ChangeTurf(/turf/simulated/floor/plasteel/cult)
+/obj/singularity/narsie/Bump(atom/A)
+	forceMove(get_turf(A))
```
Comment (phil235) [inline code/modules/power/singularity/narsie.dm]: why is this here?

Comment (ChangelingRain) [inline code/modules/power/singularity/narsie.dm]: So it doesn't get stuck on stuff; there's probably a better way to do it but basically I want it to go through walls and shit

Comment (phil235) [inline code/modules/power/singularity/narsie.dm]: please test that this doesn't cause an infinite loop.

Comment (ChangelingRain) [inline code/modules/power/singularity/narsie.dm]: forceMove() calls Crossed() but not bump and yeah it's okay, no loops happened in testing.
